sops/__init__.py

```diff
@@ -820,6 +840,10 @@ def encrypt(value, key, aad=b'', stash=None, digest=None):
     if digest:
         digest.update(value)
 
+    # if we don't want to encrypt, then return the value
+    if unencrypted:
+        return value
+
```

Comment:
This was placed in here to handle `digest.update` since it should still be part of the MAC. Did we want to relocate `digest` computation outside of `encrypt` altogether?

We could also consider resolving this by renaming `encrypt` to something like `serialize` or `dumps`